GageRnR/application.py

```diff
@@ -83,7 +83,7 @@ def __init__(self, argv=None):
     def check(self):
         if not os.path.isfile(self.file):
             raise FileNotFoundError(self.file)
-        checkIntegerList("Strucuture", self.structure, 1)
```

Comment:
This needs to be changed on line 30 as well inorder to match. A unit test should have picked that up...

Reply:
Interesting that it didn't